packages/checkbox-group/stories/index.stories.mdx

```diff
@@ -0,0 +1,336 @@
+        ></lion-checkbox>
+        <lion-checkbox
+          name="scientists[]"
+          label="Francis Bacon"
+          .choiceValue=${'Francis Bacon'}
+        ></lion-checkbox>
+        <lion-checkbox
+          name="scientists[]"
+          label="Marie Curie"
+          .choiceValue=${'Marie Curie'}
+        ></lion-checkbox>
+      </lion-checkbox-group>
+      <button @click="${() => validate()}">Validate</button>
+    `;
+  }}
+</Story>
+
+```js
+import { Required } from '@lion/validate';
+
+const validate = () => {
+  const checkboxGroup = document.querySelector('#scientistsGroup');
+  checkboxGroup.submitted = !checkboxGroup.submitted;
+};
+```
+
+```html
+<lion-checkbox-group
+  id="scientistsGroup"
+  name="scientistsGroup"
+  label="Favorite scientists"
+  .validators=${[new Required()]}
+>
+  <lion-checkbox
+    name="scientists[]"
+    label="Archimedes"
+    .choiceValue=${'Archimedes'}
+  ></lion-checkbox>
+  <lion-checkbox
+    name="scientists[]"
+    label="Francis Bacon"
+    .choiceValue=${'Francis Bacon'}
+  ></lion-checkbox>
+  <lion-checkbox
+    name="scientists[]"
+    label="Marie Curie"
+    .choiceValue=${'Marie Curie'}
+  ></lion-checkbox>
+</lion-checkbox-group>
+<button @click="${() => validate()}">Validate</button>
+```
+
+
+## Validation advanced
+
+Below is a more advanced validator on the group that evaluates the children checkboxes' checked states.
+
+<Story name="Validation Advanced">
+  {() => {
+    class HasMinTwoChecked extends Validator {
+      constructor(...args) {
+        super(...args);
+        this.name = 'HasMinTwoChecked';
+      }
+      execute(value) {
+        let hasError = false;
+        const selectedValues = value['scientists[]'].filter(v => v.checked === true);
+        if (!(selectedValues.length >= 2)) {
+          hasError = true;
+        }
+        return hasError;
+      }
+      static async getMessage() {
+        return 'You need to select at least 2 values.';
+      }
+    }
+    const validate = () => {
+      const checkboxGroup = document.querySelector('#scientistsGroup');
+      checkboxGroup.submitted = !checkboxGroup.submitted;
+    };
+    return html`
+      <lion-checkbox-group
+        id="scientistsGroup"
+        name="scientistsGroup"
+        label="Favorite scientists"
+        help-text="You should have at least 2 of those"
+        .validators=${[new Required(), new HasMinTwoChecked()]}
+      >
+        <lion-checkbox
+          name="scientists[]"
+          label="Archimedes"
+          .choiceValue=${'Archimedes'}
+        ></lion-checkbox>
+        <lion-checkbox
+          name="scientists[]"
+          label="Francis Bacon"
+          .choiceValue=${'Francis Bacon'}
+        ></lion-checkbox>
+        <lion-checkbox
+          name="scientists[]"
+          label="Marie Curie"
+          .choiceValue=${'Marie Curie'}
+        ></lion-checkbox>
+      </lion-checkbox-group>
+      <button @click="${() => validate()}">Validate</button>
```

Comment:
The validate buttons seem to be not working (both stories).

Reply:
Works for me now, check again please? :3

Reply:
Fixed, was broken in docs mode because both validation examples used the same ID